utilities_common/module.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+#
+# module_base.py
+#
+# Utility helper for modules within SONiC
+
+from sonic_py_common import logger
```

Comment:
@vvolam can you use SysLogger? https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/syslogger.py as logger has inherent limitation of polluting the global SYSLOG_IDENTIFIER

Reply:
@prgeor Modified it. But everywhere in the code, logger is used.